src/main/java/core/model/player/Player.java

```diff
@@ -1182,46 +1186,57 @@ public int getTSI() {
 //		return HOVerwaltung.instance().getModel().getEPV().getPrice(data);
 //    }
 
-    /**
-     * Setter for property m_sName.
-     *
-     * @param m_sName New value of property m_sName.
-     */
-    public void setName(java.lang.String m_sName) {
-        this.m_sName = m_sName;
+
+    public void setFirstName(java.lang.String m_sName) {
+        this.m_sFirstName = m_sName;
     }
 
-    /**
-     * Getter for property m_sName.
-     *
-     * @return Value of property m_sName.
-     */
-    public java.lang.String getName() {
-        return DBManager.deleteEscapeSequences(m_sName);
+    public java.lang.String getFirstName() {
+        return DBManager.deleteEscapeSequences(m_sFirstName);
+    }
+
+    public void setNickName(java.lang.String m_sName) {
+        this.m_sNickName = m_sName;
+    }
+
+    public java.lang.String getNickName() {
+        return DBManager.deleteEscapeSequences(m_sNickName);
+    }
+
+    public void setLastName(java.lang.String m_sName) {
+        this.m_sLastName = m_sName;
     }
 
     public java.lang.String getLastName() {
-        String lastName = this.getName();
-        return lastName.substring(lastName.lastIndexOf(" ") + 1);
+        return DBManager.deleteEscapeSequences(m_sLastName);
     }
 
+
+
     /**
      * Getter for shortName
-     *
-     * @return returns the fist letter of the first Name + a "." and the last name
-     * eg: James Bond = J. Bond, unless the string is already empty — in which
-     * case it is deemed short enough, and returned.
+     * eg: James Bond = J. Bond
+     * Nickname are ignored
      */
     public String getShortName() {
-        String fullName = getName();
 
-        if (StringUtils.isEmpty(fullName)) {
-            return fullName;
-        } else {
-            String initial = fullName.substring(0, 1);
-            String lastName = getLastName();
-            return initial + ". " + lastName;
+        if (getFirstName().isEmpty())
+        {
+            return getLastName();
+        }
+        return getFirstName().substring(0, 1) + ". " + getLastName();
```

Comment:
Out of curiosity, how does this look for players with Asian names (incl. transliteration), e.g. [this one](https://www84.hattrick.org/Club/Players/Player.aspx?playerId=394616229)?  Could this be an issue for l10n?

(I could try to recruit one if you don't have one)

Reply:
I have one ...
So currently in dev version it is broken

![image](https://user-images.githubusercontent.com/1136496/79305227-48196d00-7ef3-11ea-90b5-0beb4ff0a645.png)

with this patch it will look like :

![image](https://user-images.githubusercontent.com/1136496/79305275-5ebfc400-7ef3-11ea-993e-64480a31a0e3.png)